.ci/no-exception-test.sh

```diff
@@ -60,7 +60,7 @@ openjdk14-with-checks-nonjavadoc-error)
   cd .ci-temp/contribution/checkstyle-tester
   cp ../../../.ci/openjdk-projects-to-test-on.config openjdk-projects-to-test-on.config
   sed -i '/  <!-- Filters -->/r ../../../.ci/openjdk14-excluded.files' checks-nonjavadoc-error.xml
-  export MAVEN_OPTS="-Xmx2048m"
+  export MAVEN_OPTS="-Xmx7168m"
```

Comment:
We can get by with less memory than this, but execution is faster with increased max heap size.

Reply:
7 gigs? Are you sure machines even have that much? Some people could probably not run that locally since they have that much memory.

Reply:
Increased memory usage is an unfortunate side effect of ANTLR4; it caches DFA states to help speed up parsing. On "normal" sized repos, memory usage is increased, but not greatly. However, when we parse 50,000+ files, there is a noticeable increase in memory usage. I have a couple of tricks (clearing DFA cache, two stage parsing), that can improve the performance of ANTLR4 on gigantic repos like this, but then performance (speed/ time to parse) suffers across the board for all repos. 

That said, we can get by on openjdk14 with 6 GB, but after a few tries, I noticed that parsing is a bit faster with 7 GB.

Reply:
I am more concerned that this can't be run locally as most people may not have 7 gigs available.

For now we can ignore this.

Reply:
I am fine with this update, not ideal, but we can improve later